std/experimental/allocator/building_blocks/aligned_block_list.d

```diff
@@ -0,0 +1,629 @@
+// Written in the D programming language.
+/**
+Source: $(PHOBOSSRC std/experimental/allocator/building_blocks/_aligned_block_list.d)
```

Comment:
Preventing Ddoc's auto-hightlighting is no longer required - https://github.com/dlang/dlang.org/pull/2307